src/classes/Directory.js

```diff
@@ -84,7 +84,7 @@ class Directory {
           `Path ${this.dirType.getFolderName()} was invalid!`
         )
       }
-      return {}
```

Comment:
Was there any reason as to why we returned an empty object previously?

Reply:
great qn, may i refer you to @alexanderleegs

Reply:
Probably a bug in the past! We should standardise the return types